packages/core/src/components/graphs/pie.ts

```diff
@@ -82,6 +82,12 @@ export class Pie extends Component {
 			.sort(null)
 			.padAngle(Configuration.pie.padAngle);
 
+		// set sort function from options
+		const sortFunction = options?.pie?.sortFunction;
+		if (sortFunction) {
+			pieLayout.sort(sortFunction);
+		}
+
```

Comment:
We  should add the default sort function here https://github.com/carbon-design-system/carbon-charts/blob/8f3f8fb112277fb3fa609273dcbd27fc39a39a7d/packages/core/src/configuration.ts#L375-L383

That way there should be no need to have the if-statement